CONTRIBUTING.md

```diff
@@ -93,6 +87,35 @@ on [Submitting a patch](#submitting-a-patch) for details.
 
 [Back to TOC](#table-of-contents)
 
+## Our pledge to the community
+
+Kong Gateway is not only an awesome open source project, but also part
+of the product offering of Kong Inc.  We have a large team of product
+people, software developers, testers and release engineers working on
+Kong Gateway.  We make many of the enhancements to Kong Gateway in the
+Community Edition, and thus they become available to the open source
+community even before our commercial customers get them.
+
+Recognizing that we operate as a commercial entity, we face the
+challenge of balancing our commercial interests with the desire to
+accommodate and support open source users and contributors
+effectively.
```

Comment:
I like the professional and honest wording of this paragraph however to me personally it stands out a little bit from the rest of this doc where we have phrasings like: _"We welcome contributions of all kinds"_ or _"tasks are noble and worthy"._ 

I wonder if changing the order of sentences makes it more aligned with the doc sentiment. What do you think of this?:

> We are thrilled to work with the community and we have a strong desire to accommodate and support open source users and contributors effectively. However since we also operate as commercial entity we face the bitter challenge of balancing community work with the interest of Kong Inc.

Reply:
re: "We make many of the enhancements to Kong Gateway in the
Community Edition, and thus they become available to the open source
community even before our commercial customers get them" <<-- this isn't strictly true is it? we often release a new EE version before the OSS version don't we? suggest we remove this (and it also conflicts with what we tell enterprise customers....). AFAIK we either release OSS/EE at the same time OR EE is slightly ahead of OSS

Reply:
@nowNick I'm going to tone down the invitation to contribute a bit.  It comes from six years ago, when Kong was primarily managed as Open Source project, and I agree that it does not quite align with what we're now pledging.

@teb510 Most enhancements become available in the open source repository first, but I understand your point and will change the wording.

Reply:
@teb510 It now reads "We make many of the enhancements to Kong Gateway in the
Community Edition, so our open source community directly benefits from
the commercial work that we do."